piksi_tools/stm_unique_id.py

```diff
@@ -19,22 +19,31 @@
 
 class STMUniqueID:
 
-  def __init__(self, link, version):
+  def __init__(self, link):
+    self.heartbeat_received = False
     self.unique_id_returned = False
+    # SBP version is unset in older devices.
+    self.sbp_version = 0
     self.unique_id = None
     self.link = link
-    self.version = version
+    link.add_callback(self.receive_heartbeat, SBP_MSG_HEARTBEAT)
```

Comment:
Oh nice. I was thinking it would get passed when initialized after receiving the heartbeat but this is way nicer.